_guides/getting_started.md

```diff
@@ -10,31 +10,60 @@ body_class: tutorial
 ```
 
-There is quite a bit going on here. The first number in each line is a label. A `0` label corresponds to no roof-replacement, while a `1` label corresponds to a roof-replacement. The bar `|` separates label related data (what we want to predict) from features (what we always know). The features in the 1st line are `price`, `sqft`, `age`, and `2006`. Each feature may have an optional `:<numeric_value>` following it or, if the value is missing, an implied value of `1`. By default, Vowpal Wabbit hashes feature names into in-memory indexes unless the feature names themselves are positive integers. In this case, the first 3 features use an index derived from a hash function while the last feature uses index 2006 directly. Also the 1st 3 features have explicit values (`.23`, `.25`, and `.05` respectively) while the last, `2006` has an implicit default value of 1.
+Vowpal Wabbit hashes feature names into in-memory indexes by default unless the feature names are positive integers. In the first line of the `house_dataset` example, the first three features use an index derived from a hash function while the last feature uses index `2006` directly. Also, the first three features have explicit values (`.23`, `.25`, and `.05` respectively) while the last, `2006` has an implicit default value of `1`.
+
+```
+0 | price:.23 sqft:.25 age:.05 2006
+```
+
+- The first number in each line is a label. 
+- A `0` label corresponds to no roof-replacement, while a `1` label corresponds to a roof-replacement. 
+- The bar `|` separates label related data (what we want to predict) from features (what we always know). 
+- The features in the first line are `price`, `sqft`, `age`, and `2006`. Each feature may have an optional `:<numeric_value>` following it or, if the value is missing, an implied value of `1`. 
 
-The next example, on the next line, is similar, but the label information is more complex. The `1` is the label indicating that a roof-replacement is required. The `2` is an optional importance weight which implies that this example counts twice. Importance weights come up in many settings. A missing importance weight defaults to 1. `'second_house` is the tag, it is used elsewhere to identify the example.
+The label information for the second line is more complex than the first: 
 
-The 3rd example is straightforward, except there is an additional number: `0.5` following the importance weight, in the label information. This is an initial prediction. Sometimes you have multiple interacting learning systems and want to be able to predict an offset rather than an absolute value.
+`1 2 'second_house | price:.18 sqft:.15 age:.35 1976`
+
+The `1` is the label indicating that a roof replacement is required. 
+The `2` is an optional _importance weight_, which implies that this example counts twice. The importance weight comes up in many settings. 
+A missing importance weight defaults to 1. `'second_house` is the tag, it is used elsewhere to identify the example.
```

Comment:
I just realized that the wording of `it is used elsewhere to identify the example` is quite vague. It does show up later in this tutorial so shouldn't we setup the right expectation?

```suggestion
A missing importance weight defaults to 1. `'second_house` is the tag, it will show up later to help you identify the example.
```